### DISCUSSION TITLE
Ajout d'une commande pour les pets

### DISCUSSION THREAD

**Discussion by DarkRaptor8** [Crownicles/Crownicles#779] [Idées acceptées / déjà implémentées]
Je propose qu'il est un commande nous permettant de voir le niveau d'affection avec un pet par exemple :  Niveau d'affection avec (Nom du pet) 8/25.Pour savoir quand le pet augmentra de niveau d'affection et lorsqu'il augmente on es , De DraftBot ⬆(Nom du pet) a augmenté de niveau d'affection (Nom du pet) est désormais : Apprivoiser

**Comment by As-Nodt** [comment]
Bonsoir, 

Il me semble avoir déjà proposé cette suggestion avec une autre formulation. Il n'est pas nécessaire de créé une nouvelle commande pour voir la progression du dressage de nos pets, il suffirait de prendre exemple de la commande !g qui nous affiche le level et l'avancement en % de nos guildes et de créer simplement une jauge, une barre de progression sur l'affichage de la commande !pet. Cela serait vraiment pratique, j'espère que ça sera accepté 🤞

**Reply by angels-hub** [reply]
avec les différents paliers ?

**Comment by Pietagorh** [comment]
Comme on l'a sûrement déjà dit la première fois que t'as fait la suggestion, on ne veut pas faire un affichage aussi précis, pour pas que les pets paraissent trop "robotiques" x)

**Reply by angels-hub** [reply]
une barre de progression est un bon compromis je penses, à condition de pas mettre de % à côté

**Reply by Pietagorh** [reply]
Bah la barre de progression est composée de 20 caractères, donc t'es au courant de la mentalité du pet à 5% près
C'est proche

**Reply by angels-hub** [reply]
honnetement les premiers stades sont tellement proches que on verra pas la différence mais ca peut aider pour le dernier stade ou genre c'est vraiment long

**Comment by BastLast** [comment]
Pr info nous avons développé un moyen de connaître l'amour d'un pet avec plus de précision, ça arrivera bientôt !

**Reply by Pietagorh** [reply]
C'est vrai j'oubliais

**Comment by DarkRaptor8** [comment]
J'ai fais un carton

**Comment by BastLast** [comment]
je déplace cette suggestion dans accepté car une fonctionnalités similaire a été implémentée.
